evalml/utils/gen_utils.py

```diff
@@ -261,9 +261,11 @@ def _rename_column_names_to_numeric(X, flatten_tuples=True):
         return pd.DataFrame(X)
 
     X_renamed = X.copy()
+    logical_types = X.ww.logical_types
     if flatten_tuples and (len(X.columns) > 0 and isinstance(X.columns, pd.MultiIndex)):
```

Comment:
If I'm reading this correctly, it seems like we no longer ever set `flatten_tuples` to False. In that case, are we ok to just remove the argument entirely?